src/Microsoft.AspNetCore.Http.Features/IHttpRequestFeature.cs

```diff
@@ -49,6 +49,15 @@ public interface IHttpRequestFeature
         string QueryString { get; set; }
 
         /// <summary>
+        /// The request target as it was sent in the HTTP request. This property contains the
+        /// escaped path and full query, as well as other request targets such as * for OPTIONS
```

Comment:
"escaped path"

Does this mean that the precent encoded characters in path are unescaped or raw? I think the word escaped implies that the server did some sort of escaping.